src/main/java/rs117/hd/lighting/EquipmentLight.java

```diff
@@ -0,0 +1,69 @@
+package rs117.hd.lighting;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import net.runelite.api.ItemID;
+import static net.runelite.api.ItemID.*;
+import rs117.hd.lighting.LightManager.LightType;
+import rs117.hd.lighting.LightManager.Alignment;
+
+@AllArgsConstructor
+@Getter
+enum EquipmentLight
+{
+        FIRE_CAPE(75, Alignment.BACK, 135, 8f, rgb(198, 156, 74), LightType.PULSE, 2100, 10, ItemID.FIRE_CAPE, FIRE_CAPE_10566),
+        FIRE_MAX_CAPE(75, Alignment.BACK, 135, 8f, rgb(198, 156, 74), LightType.PULSE, 2100, 10, ItemID.FIRE_MAX_CAPE, FIRE_MAX_CAPE_21186),
+        INFERNAL_CAPE(75, Alignment.BACK, 135, 8f, rgb(133, 64, 0), LightType.PULSE, 2100, 10, ItemID.INFERNAL_CAPE, INFERNAL_CAPE_21297, INFERNAL_CAPE_23622),
+        INFERNAL_MAX_CAPE(75, Alignment.BACK, 135, 8f, rgb(133, 64, 0), LightType.PULSE, 2100, 10, ItemID.INFERNAL_MAX_CAPE, INFERNAL_MAX_CAPE_21285),
+    ;
+
+    private final int[] id;
+    private final int height;
+    private final Alignment alignment;
+    private final int size;
+    private final float strength;
+    private final int rgb;
+    private final LightType lightType;
+    private final float duration;
+    private final float range;
+
+    EquipmentLight(int height, Alignment alignment, int size, float strength, int rgb, LightType lightType, float duration, float range, int... ids)
+    {
+        this.height = height;
+        this.alignment = alignment;
+        this.size = size;
+        this.strength = strength;
+        this.rgb = rgb;
+        this.lightType = lightType;
+        this.duration = duration;
+        this.range = range;
+        this.id = ids;
+    }
+
+    private static final Map<Integer, EquipmentLight> LIGHTS;
+
+    static
+    {
+        ImmutableMap.Builder<Integer, EquipmentLight> builder = new ImmutableMap.Builder<>();
+        for (EquipmentLight equipmentLight : values())
+        {
+            for (int id : equipmentLight.id)
+            {
+                builder.put(id + 512, equipmentLight);
+            }
+        }
+        LIGHTS = builder.build();
+    }
+
+    static EquipmentLight find(int id)
+    {
+        return LIGHTS.get(id);
+    }
+
+    private static int rgb(int r, int g, int b)
+    {
+        return (r << 16) | (g << 8) | b;
+    }
+}
```

Comment:
You're missing a newline here

Reply:
Also above here, is it worth/easy to add config for other bits of equipment?

The ones that come to mind are

- https://oldschool.runescape.wiki/w/Bug_lantern
- https://oldschool.runescape.wiki/w/Hallowed_focus

Reply:
The more I think about it, the more pieces of equipment I can think of. Makes sense if you want to leave this PR as just infra and then a second one adding all the equipment

Reply:
it's generally pretty easy just got to setup the positioning and brightness etc.  It works the same as adding lights for objects, npcs etc. just a bit outside of the scope of both the issue and the amount of time I had available to spend on writing this.